webrtc.html

```diff
@@ -7841,6 +7852,28 @@ <h2>Attributes</h2>
               negotiation is complete, the <code>mid</code> value may be null.
               After rollbacks, the value may change from a non-null value
               to null.</p>
+              <p>On getting, the attribute MUST return the value of the
+              <a>[[\Mid]]</a> slot.</p>
+              <p>Whenever <span data-jsep=
+              "subsequent-offers subsequent-answers">[[!JSEP]]</span> says to
+              modify the "mid property", the user agent MUST queue a task that
+              runs the following steps:</p>
```

Comment:
I think this is the wrong place to put this algorithm. This decouples it from setting an RTCSessionDescription and it becomes unclear whether to update the mid before or after firing events etc. But is this algorithm even needed, we're already updating the mid inside of the task that was queued for SLD/SRD. Let's not queue two tasks.